keyboards/hhkb/keymaps/mjt/keymap.c

```diff
@@ -2,6 +2,7 @@
  * default HHKB Layout
  */
 #include QMK_KEYBOARD_H
+#include "dynamic_macro.h"
```

Comment:
I'm not a big fan of changing user keymaps. Will these continue to work without being changed, or will this PR cause them to start erroring out?